.eslintignore

```diff
@@ -1,9 +1,11 @@
 # Ignore 3rd party files
 node_modules
+iframeResizer.contentWindow.min.js
```

Comment:
We could consider including `*.min.js` in "generated files"

Reply:
As discussed, let's add another section for "Ignore minified files" as it's not generated by us and it's probably better not to make future developers assume something different :)